examples/font.amp.html

```diff
@@ -0,0 +1,107 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>Font example</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async custom-element="amp-font" src="https://cdn.ampproject.org/v0/amp-font-0.1.js"></script>
+  <style>body {opacity: 0}</style><noscript><style>body {opacity: 1}</style></noscript>
+  <style type="text/css" media="screen">
+    @font-face {
+      font-family: 'Comic AMP';
+      font-style: normal;
+      font-weight: 400;
+      src: url(fonts/ComicAMP.ttf) format('truetype');
+    }
+
+    @font-face {
+      font-family: 'Comic AMP Bold';
+      font-style: italic;
+      font-weight: 600;
+      src: url(fonts/ComicAMPBold.ttf) format('truetype');
+    }
+  </style>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <style type="text/css" media="screen">
```

Comment:
We allow only one `<style>` tag and it must be `<style amp-custom>`, besides opacity boilerplate of course.